docs-cg.md

```diff
@@ -0,0 +1,67 @@
+# W3C Documentation CG Charter
+
+## Mission
+
+The mission of the W3C Documentation Community Group (W3C Docs CG) is to ensure web developers and designers worldwide have the best information available so they can build things on the web platform. The Docs CG focuses on the art of technical writing and information architecture across the Web ecosystem (W3C, WHATWG, ECMA, IETF, etc.).
+
+## Scope of work
+
+This Community will:
+
+- Plan and write new documentation for web platform technologies.
+  - Make sure reference docs (“the book of web platform features”) are complete.
+  - Identify needs for Guides, How-Tos, and Tutorials and create them.
+- Work with specification authors to define best practices for web developer docs, code examples, and information architecture.
+- Maintain and keep documentation up-to-date using modern approaches like docs-as-code and develop tools to create feedback loops between standards and web developer documentation.
+- Collaborate with other horizontal groups such as WebDX, Security, Privacy, Accessibility, Internationalization, Sustainability and integrate their findings and best practices into docs.
+- Create a map of documentation sources for web developers.
+  - Identify reliable sources of information.
+  - Invite them to be part of the conversations in the CG.
+- Drive adoption (or deprecation!) of web platform technologies.
+  - Identify web platform features where sparse docs prevent adoption and understanding.
+  - Interpret developer surveys (such as the [MDN short survey on web Developer’s experience with Web Security](https://github.com/web-platform-dx/developer-research/blob/main/mdn-short-surveys/2023-05-15-security-dx/interpretation.md) and understand the role of documentation in identified shortcomings.
+- Coordinate horizontal efforts to get outdated docs updated by, e.g. helping reach out to the right docs owners.
+- Act as a hub for when docs-related efforts/changes/projects appear, so people can get these amplified/coordinated with the relevant stakeholders.
+
+## Key deliverables
+
+- Write and maintain docs in [mdn/content](https://github.com/mdn/content). See the [backlog](https://openwebdocs.github.io/web-docs-backlog/) for missing reference pages.
+- Identify best practices for web developer documentation.
+- Drive adoption and understanding of web technologies.
+- Act as a hub to coordinate documentation projects.
+
+## Out of scope
+
+The group may produce Community Group Reports within the scope of this charter but not Specifications.
+
+## Dependencies or liaisons
+
+- Groups
+  - Open Web Docs
+  - Web Developer Experience (WebDX) Community Group
+  - Webref & spec tooling maintainers
```

Comment:
The [Spec Editors CG](https://www.w3.org/community/speced-cg/) maintains some of these tools and probably deserves to appear in the list. But then other spec authoring tools come to mind, including [Wattsi](https://github.com/whatwg/wattsi), used for the HTML spec, specific ones used in TC39 and IETF, or [SpecTec](https://webassembly.org/news/2025-03-27-spectec/) used for WebAssembly.